### PR TITLE
acknowledge the potential existence of other clients aside from foo's client

### PR DESCRIPTION
**Pull request by SMOLKEYS**
If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [X] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [ ] ~~I have ensured that my code compiles, if applicable.~~
- [ ] ~~I have ensured that any new features in this PR function correctly in-game, if applicable.~~

### CODE REVIEW

**Comment by buthed010203** [comment]
👍 inclusivity. Fwiw i think most people who use foo's are already smart enough to not report issues here or think its a mod anyways

**Comment by Anuken** [comment]
Too wordy.

**Comment by buthed010203** [comment]
"Enaure your game is fully vanilla/unmodded"?
